watson_embed_model_packager/setup_build_config.py

```diff
@@ -487,21 +494,29 @@ def get_models_from_local_dir(model_dir_path: str) -> List[ModelInfo]:
     for zip_file in os.listdir(model_dir_path):
         path_to_zip_file = os.path.join(model_dir_path, zip_file)
         if os.path.isfile(path_to_zip_file) and path_to_zip_file.endswith(".zip"):
-            dest_path = path_to_zip_file.replace(".zip", "")
             log.debug(
-                "Extracting zip file %s into folder %s", path_to_zip_file, dest_path
+                "Extracting zip file %s into location %s",
+                path_to_zip_file,
+                model_dir_path,
             )
             with zipfile.ZipFile(path_to_zip_file, "r") as zip_ref:
-                zip_ref.extractall(dest_path)
+                zip_ref.extractall(model_dir_path)
                 zip_ref.close()
+            log.debug(
+                "%s now has child dir %s", model_dir_path, os.listdir(model_dir_path)
+            )
 
     for child_dir in os.listdir(model_dir_path):
-        full_model_dir = os.path.join(model_dir_path, child_dir)
-        if os.path.isdir(full_model_dir) and "config.yml" in os.listdir(full_model_dir):
-            model = get_model_info_from_local_config_yml(
-                child_dir, os.path.join(full_model_dir, "config.yml")
-            )
-            local_models.append(model)
+        if not child_dir.startswith("__MACOSX"):
```

Comment:
luhmayo

Was this actually a problem? It would be kinda bad if the later checks for a config.yml failed to filter out these os-specific backup folders :(